packages/manager/apps/public-cloud/src/components/beta-warning/beta-warning.controller.js

```diff
@@ -0,0 +1,20 @@
+import illustration from './beta-warning.png';
+
+const LOCALSTRORAGE_KEY = 'ovhManagerPublicCloud_hideBetaWarning';
```

Comment:
I'm not convinced about using both camelCase and snakeCase.

For instance:
```suggestion
const LOCALSTRORAGE_KEY = 'OVH_MANAGER_PUBLIC_CLOUD_HIDE_BETA_WARNING';
```

Reply:
```suggestion
const LOCALSTORAGE_KEY = 'ovhManagerPublicCloud_hideBetaWarning';
```
also :)